openzwave-sys/build.rs

```diff
@@ -1,13 +1,23 @@
 extern crate gcc;
+extern crate pkg_config;
 
 fn main() {
+    let libopenzwave = pkg_config::Config::new()
+        .statik(true)
```

Comment:
Maybe not static. Actually I don't think I get static linkage anyway. I'll remove this line.